### ISSUE TITLE
Comment Connetion Query Args: where: {orderby: ...}

### ISSUE THREAD

**Issue by tonimain** [wp-graphql/wp-graphql#167]
Found a bug while writing this tests WIP.
The function orderBy is causing an error where the query returns:
```
SyntaxError: Unexpected token < in JSON at position 0
```

This is the query:
```
query {
  comments(where: {orderby: COMMENT_DATE}) {
    edges {
      node {
	approved
        id
        author {
         email
          url
        }
        commentId
        content
        type
        
      }
    }
  }
}
```

#61

**Comment by jasonbahl** [comment]
@tonimain thanks for reporting. I see the issue and can get this corrected soon.

**Comment by jasonbahl** [comment]
This was fixed by #174
